### ISSUE TITLE
RCE Bypass: Via the `trap` command

### ISSUE THREAD

**Issue by HackingRepo** [coreruleset/coreruleset#4421]
Hi, Team

While testing the Coraza WAF sandbox, I noticed that the following payload passes without detection or blocking

```rust
curl -ig -H "x-format-output: txt-matched-rules" -H "x-crs-paranoia-level: 3" --data-urlencode "q=;trap 'rm -rf file.txt' SIGEXIT" "https://sandbox.coreruleset.org/"
```

and the response is

```rust
HTTP/1.1 200 OK
Date: Fri, 19 Dec 2025 06:49:31 GMT
Content-Type: text/plain
Transfer-Encoding: chunked
Connection: keep-alive
X-Unique-ID: aUT1e-f-Hh3gitdLV0D4NQAAAEg
x-backend: invalid, fallback to apache-latest
x-crs-last-commit: none
```

Expected Behavior:

- CRS should block the `;trap` command

The impact of this:
- That is Blind RCE unlike the normal RCE, The attacker see no output, But attacker can still use it to execute commands that delete a file like `rm` and write to files or backdoors when the action triggered when the web server or server stop the file will be deleted like in reboot or outage.
